PSKoans/Koans/Constructs and Patterns/AboutStringBuilder.Koans.ps1

```diff
@@ -60,12 +60,13 @@ Describe 'System.Text.StringBuilder' {
 
         It 'can append new lines' {
             $StringBuilder = [System.Text.StringBuilder]::new()
-            $StringBuilder.AppendLine('Hello!')
-            $StringBuilder.AppendLine('Goodbye!')
+            $StringBuilder.AppendLine('__')
+            $StringBuilder.AppendLine('__')
 
             $ExpectedString = @"
-__
-__
+Hello!
+GoodBye!
+
 "@
             $ExpectedString | Should -Be $StringBuilder.ToString()
```

Comment:
Even with your changes here, this assertion falls flat on Windows, thanks to how stringbuilder does its line appending.

We may need to add the carriage returns explicitly in the `$ExpectedString` conditionally only on Windows, as messy as that is. A comment explaining it is probably also warranted.